core/project_settings.cpp

```diff
@@ -1044,6 +1046,14 @@ ProjectSettings::ProjectSettings() {
 	custom_prop_info["compression/formats/gzip/compression_level"] = PropertyInfo(Variant::INT, "compression/formats/gzip/compression_level", PROPERTY_HINT_RANGE, "-1,9,1");
 
 	using_datapack = false;
+
```

Comment:
This also is core using servers, but should go fine in main.cpp too. you can call it before setting up the servers.